.gitignore

```diff
@@ -74,6 +74,9 @@ ENV/
 # PyCharm / IntelliJ
 .idea
 
+# VS Code
+.vscode
+
```

Comment:
This will remove recommended editor configuration.  Please keep this.

Reply:
Thought it was my local editor that created those.